README.md

```diff
@@ -77,6 +77,25 @@ With valgrind, you might need to increase the max stack size:
 
     $ valgrind --max-stackframe=2500000 ./exhaustive_tests
 
+Test coverage
+-----------
+
+To create a test coverage report, configure with `--enable-coverage` (use of GCC is necessary):
+
+    $ ./configure --enable-coverage
+
+Run the tests:
+
+    $ make check
+
+To create a report, `gcovr` is recommended, as it includes the coverage of all branches:
```

Comment:
nit: s/coverage of all branches/branch coverage reporting

Reply:
Done.